python/cudf/cudf/core/column/lists.py

```diff
@@ -178,6 +178,35 @@ def __init__(self, column, parent=None):
             )
         super().__init__(column=column, parent=parent)
 
+    def contains(self, key):
+        """
+        Creates a column of bool values indicating whether the specified scalar
+        is an element of each row of a list column.
+
+        Parameters
+        ----------
+        search_key : scalar
+            element being searched for in each row of the list column
+
+        Returns
+        -------
+        Column
+
+        Examples
+        --------
+        >>> s = cudf.Series([[1, 2, 3], [3, 4, 5], [4, 5, 6]])
+        >>> s.list.contains(4)
+        Series([False, True, True])
+        dtype: bool
+        """
+        # if key is not None:
+        search_key = cudf.Scalar(key).device_value
+        # else:
+        # search_key = key
```

Comment:
Remove dead code

Reply:
i haven't pushed my recent edits. will be updating shortly